src/libgrate/grate.c

```diff
@@ -152,14 +152,10 @@ void grate_bind_framebuffer(struct grate *grate, struct grate_framebuffer *fb)
 	grate->fb = fb;
 }
 
-struct host1x_bo * grate_get_front_framebuffer_bo(struct grate_framebuffer *fb)
+struct host1x_pixelbuffer * grate_get_actual_framebuffer_pixbuf(
```

Comment:
This is a small nit, but "actual" sounds a bit too abstract to me. How about using OpenGL terms (draw-buffer), and say `grate_get_draw_pixbuf` instead?